packaging/ubuntu/click/manifest.json

```diff
@@ -15,6 +15,6 @@
         }
     },
     "version": "1.0.100",
-    "maintainer": "Michael Zanetti <michael.zanetti@nymea.io>",
+    "maintainer": "nymea GmbH <michael.zanetti@nymea.io>",
```

Comment:
Makes sense. I suppose in that case the correct email would be `developer@nymea.io`

Reply:
Done.